src/modules/hip/kernel/fog.hpp

```diff
@@ -0,0 +1,323 @@
+#include <random>
```

Comment:
why this random header file needs to be included?

Reply:
I removed the random header file . Since it is not needed .